modules/admin_manual/pages/installation/manual_installation/manual_installation.adoc

```diff
@@ -37,32 +38,31 @@ This guide installs PHP 7.4
 
 === Install a Database
 
-If you do not already have installed a supported database or using an existing one, follow the
-xref:installation/manual_installation/manual_installation_db.adoc[Manual Installation Databases guide].
+If you do not already have a supported database installed, follow the
+xref:installation/manual_installation/manual_installation_db.adoc[Manual Database Installation guide].
 
 === Configure the Web Server
 
-To configure your Apache Web Server for the use with ownCloud, follow the
+To configure your Apache web server for use with ownCloud, follow the
 xref:installation/manual_installation/manual_installation_apache.adoc[Apache preparation guide].
 
 == Installation of ownCloud Binaries
 
 To install ownCloud binaries, you have to download the required package. After doing so,
-you can do the following steps manually, or use provided scripts. These scripts are benificial,
-because they also can be used for upgrading and ease the process a lot. 
+you can perform the following steps manually or use the provided scripts. These scripts are convenient
+since they can also be used for upgrading which eases the process a lot. 
  
 === Download ownCloud
 
-Before downloading the ownCloud package, change to a directory where you will save
-your file temporarily. This can be, for example `/tmp`. In further examples, we use tar archives and
-the actual `ownCloud Complete Package`. The package name for the complete package is formed the following:
+Before downloading ownCloud, change to a directory where you want to save
+the file temporarily. This can be, for example `/tmp`. In further examples, we use tar archives or the complete ownCloud. The name for the complete archive looks like this:
```

Comment:
```suggestion
the file temporarily. This can be, for example `/tmp`. In further examples, we use tar archives or the complete ownCloud package. The name for the complete archive looks like this:
```